### ISSUE TITLE
Request for Top 5 Logits at Each Output Position

### ISSUE THREAD

**Issue by lnennnn** [DAMO-NLP-SG/VideoLLaMA3#65]
Thank you for the excellent work！

I wonder if it is feasible to provide the logits for the top 5 predicted tokens at each output position, similar to the functionality implemented in the OpenAI Completion API?

Best wishes :)
